packages/jest/src/generators/configuration/configuration.spec.ts

```diff
@@ -139,6 +139,8 @@ describe('jestProject', () => {
         'jest.config.ts',
         'src/**/*.test.ts',
         'src/**/*.spec.ts',
+        'src/**/*.js',
```

Comment:
I think the larger discuss here is whether solutions-style tsconfig should be kept or not. If we remove the project references, then the `tsconfig.app.json` and `tsconfig.lib.json` can be kept as is, and imports will be picked up from `*.spec.ts` files to production files.